### ISSUE TITLE
TypeError: proto.server is not a function

### ISSUE THREAD

**Issue by cryptix** [ssbc/ssb-server#577]
```
ssb-electron-chroot-chrootenv:cryptix@crypdesk:~/scuttle-shell$ npm start

> scuttle-shell@0.1.3 start /home/cryptix/scuttle-shell
> node server.js

started sbot server v13.1.0
scuttle-shell got exit: { code: 0, signal: null }
/home/cryptix/scuttle-shell/node_modules/multiserver/compose.js:96
      return asyncify(proto.server(function (stream) {
                            ^

TypeError: proto.server is not a function
    at Object.server (/home/cryptix/scuttle-shell/node_modules/multiserver/compose.js:96:29)
    at /home/cryptix/scuttle-shell/node_modules/multiserver/index.js:31:21
    at Array.map (<anonymous>)
    at Object.server (/home/cryptix/scuttle-shell/node_modules/multiserver/index.js:30:26)
    at Immediate.setupMultiserver [as _onImmediate] (/home/cryptix/scuttle-shell/node_modules/secret-stack/index.js:173:21)
    at runCallback (timers.js:810:20)
    at tryOnImmediate (timers.js:768:5)
    at processImmediate [as _immediateCallback] (timers.js:745:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! scuttle-shell@0.1.3 start: `node server.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the scuttle-shell@0.1.3 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /tmp/buttEnv-npm-cache/_logs/2018-11-23T21_11_18_739Z-debug.log
```

inserting a `console.log(proto)` before the return gives me:

```
{ name: 'unix',
  scope: [Function: scope],
  server: null,
  client: [Function: client],
  parse: [Function: parse],
  stringify: [Function: stringify] }
```

do we need to change this maybe? https://github.com/ssbc/scuttlebot/blob/d1cec64cc29c4dca784cf7c8cdf897f9b4f724d8/plugins/unix-socket.js#L9 I'll try to add a listen / dial test soon.

**Comment by cryptix** [comment]
cc @arj03

**Comment by christianbundy** [comment]
Did that change come from https://github.com/ssbc/multiserver/pull/32?

cc: @dominictarr

**Comment by arj03** [comment]
Closed by multiserver 3.3.0
